src/components/home/ProjectCard.tsx

```diff
@@ -0,0 +1,31 @@
+import Image, { StaticImageData } from "next/image";
+
+interface ProjectCardProps {
+  backgroundImage?: StaticImageData;
+  description: string;
+  title: string;
+}
+
+const ProjectCard: React.FC<ProjectCardProps> = ({
+  backgroundImage,
+  description,
+  title,
+}) => {
+  return (
+    <div className="flex flex-col items-center rounded-2xl border-4 border-devstation-green-400 bg-devstation-green-300 p-5">
+      <div className="mb-2 flex w-3/4 items-end justify-end rounded-2xl">
+        {backgroundImage && (
+          <Image src={backgroundImage} alt={title} className="rounded-2xl" />
+        )}
+
+        <div className="absolute flex items-end justify-end px-[2px] py-[20px] text-right uppercase">
```

Comment:
px-[2px] py-[20px]
use the tailwind classes instead of using [custom_value]

Reply:
Sure, let me use `px-0.5 py-5` in that case.